corehq/ex-submodules/casexml/apps/phone/restore_caching.py

```diff
@@ -0,0 +1,75 @@
+import hashlib
+import logging
+from casexml.apps.phone.const import RESTORE_CACHE_KEY_PREFIX, ASYNC_RESTORE_CACHE_KEY_PREFIX
+from dimagi.utils.couch.cache.cache_core import get_redis_default_cache
+
+logger = logging.getLogger(__name__)
+
+
+def _restore_cache_key(domain, prefix, user_id, version, sync_log_id, device_id):
+    # to invalidate all restore cache keys, increment the number below
+    hashable_key = '8-{domain}-{prefix}-{user}-{version}-{sync_log_id}-{device_id}'.format(
```

Comment:
Yeah, good point. Honestly I picked 8 because every time I wanted to try it fresh locally i would bump it, and I ended up with 8 there.

I'll change the dashes to commas.